src/artemis/external_interaction/callbacks/fgs/zocalo_callback.py

```diff
@@ -87,25 +88,52 @@ def wait_for_results(self, fallback_xyz: Point3D) -> Point3D:
                 datacollection_group_id
             )
 
+            # Example result with two crystals:
```

Comment:
Could you remove this comment if not needed? Since you already have a similar example in the docstring of wait_for_results() in `zocal_interaction.py`, it feels a bit redundant